kura/org.eclipse.kura.core.cloud/META-INF/MANIFEST.MF

```diff
@@ -15,6 +15,7 @@ Import-Package: org.eclipse.kura;version="[1.0,2.0)",
  org.eclipse.kura.cloud.factory;version="[1.1,1.2)",
  org.eclipse.kura.command;version="[1.1,1.2)",
  org.eclipse.kura.configuration;version="[1.0,2.0)",
+ org.eclipse.kura.configuration.metatype;version="[1.0.0,2.0.0)",
```

Comment:
So that means that you can't use the Eclipse IDE anymore for editing. And it doesn't provide any technical benefit.